resources/assets/less/bem/osu-page-header.less

```diff
@@ -130,18 +130,28 @@
     .at2x-simple("/images/headers/tournament.jpg");
   }
 
-  &--wiki {
+  .wiki {
```

Comment:
elements can have multiple modifiers:
`class="osu-page-header--wiki osu-page-header--wiki-main-page"`
 then you can do:
```
  &--wiki {
    .@{top}--wiki-main-page& {
    }
  }
```
instead.

Also mixins should be declared with parentheses.